muffnn/mlp/tests/test_mlp_classifier.py

```diff
@@ -357,3 +357,44 @@ def test_sample_weight(make_dataset_func, dataset_kwargs):
         lambda: MLPClassifier(n_epochs=30, random_state=42,
                               keep_prob=0.8, hidden_units=(128,))
     )
+
+
+def test_prediction_gradient():
+    """Test computation of prediction gradients."""
+    # Binary classification
+    n_classes = 1
+    mlp = MLPClassifier(n_epochs=100, random_state=42, hidden_units=(5,))
+    X, y = make_classification(
+        n_samples=1000, n_informative=n_classes, n_redundant=0,
+        n_classes=n_classes, n_clusters_per_class=1, shuffle=False)
+    mlp.fit(X, y)
+    grad = mlp.prediction_gradient(X)
+    grad_means = grad.mean(axis=0)
+    assert grad.shape == X.shape
+    # Check that only the informative feature has a large gradient.
+    assert np.abs(grad_means[0]) > 1.
+    for m in grad_means[1:]:
+        assert np.abs(m) < 0.5
```

Comment:
Why 0.5?

Reply:
This was fairly arbitrary.  Let me write a comment here and see if you think it makes snse